packages/turf/test.ts

```diff
@@ -4,10 +4,10 @@ import { glob } from "glob";
 import test from "tape";
 import camelCase from "camelcase";
 import documentation from "documentation";
-import * as turf from "./index";
+import * as turf from "./index.js";
 
 // Helpers
-const directory = path.join(__dirname, "..");
+const directory = path.join("..");
```

Comment:
This is maybe a little extra sketchy and might be worth using `import.meta` since its reaching into other packages from here.

Reply:
Have reintroduced this __dirname reference.